src/app/app-routing.module.ts

```diff
@@ -14,14 +14,15 @@ import {
   PROFILE_MODULE_PATH,
   ADMIN_MODULE_PATH,
   BITSTREAM_MODULE_PATH,
-  INFO_MODULE_PATH
+  INFO_MODULE_PATH, FORBIDDEN_PATH
```

Comment:
Tiny thing...just for readability, it might be nice to put this new constant on it's own line.

Reply:
In general we don't look at the imports. Our IDE manages them automatically and even keeps them hidden by default these days. I understand a reviewer asking to remove unused imports, or not to reorder unnecessarily as that makes PRs seem bigger than they actually are. 

However manually going over the imports the IDE has added to determine whether an import should go on a new line or not doesn't seem like a very good use of a developer's time.

Reply:
@artlowel : makes sense. I wasn't sure if this was something that is an IDE configuration that could be tweaked.  But, no big deal to let these imports align however the IDE decides to do them.  So, I agree with you, this is not really worth the time, so I'll refrain from these comments in the future ;)